### ISSUE TITLE
On using redux-form with redux-thunk

### ISSUE THREAD

**Issue by keithamoss** [ealgis/ealgis#60]
tl;dr It's not straight forward and redux-form's API has gone through a lot of changes, so most answers are now out of date or subtly not quite right :(

Original thread we started from: https://stackoverflow.com/questions/34142678/make-server-validation-using-redux-form-and-fetch-api

Related redux-form issues:

- https://github.com/erikras/redux-form/issues/2569
- https://github.com/erikras/redux-form/issues/2481
- https://github.com/erikras/redux-form/issues/2609
- https://github.com/erikras/redux-form/issues/2269
- https://github.com/erikras/redux-form/issues/1898

redux-form issues to monitor:

- https://github.com/erikras/redux-form/issues/2068

On using react-router with redux-form: http://stackoverflow.com/questions/38882157/accessing-this-context-from-onsubmitsuccess-of-redux-form

**Comment by keithamoss** [comment]
Solved this
